src/systemTest/java/com/flightstats/hub/resilient/HelmReleaseTest.java

```diff
@@ -0,0 +1,35 @@
+package com.flightstats.hub.resilient;
+
+import com.flightstats.hub.resilient.helm.ReleaseDelete;
+import com.flightstats.hub.resilient.helm.ReleaseInstall;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.After;
+import org.junit.Test;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+@Slf4j
+public class HelmReleaseTest extends ResilientBaseTest {
+
+    private String releaseName = "ddt-" + generateRandomString();
+    @Inject
+    private ReleaseInstall releaseInstall;
+    @Inject
+    private ReleaseDelete releaseDelete;
+
+    @Inject
+    @Named("helm.chart.path")
+    private String chartPath;
+
+    @Test
+    public void test() {
+        this.releaseInstall.install(this.releaseName, this.chartPath);
```

Comment:
If this is going to be a long-lived test, it may make sense to add some sort of assertion that resource creation gets rolling. Does install wait for all the services and whatnot to return good values from their readiness probes? If not, we'll probably need to figure out how to check that various services/sites start up before we start running the rest of our tests.

Reply:
oh yeah. There is a class ReleaseInstall that ensures this. Again helm package for hub is WIP. Some of the resources fail to instantiate. Rich is working on sequencing the resources instantiation. This will be WIP too.